src/main/scala/org/ergoplatform/http/api/WalletApiRoute.scala

```diff
@@ -286,31 +286,20 @@ case class WalletApiRoute(readersHolder: ActorRef, nodeViewActorRef: ActorRef, e
 
   def transactionsR: Route = (path("transactions") & get & txParams) {
     case (minHeight, maxHeight, minConfNum, maxConfNum) =>
-      if ((minHeight > 0 || maxHeight < Int.MaxValue) && // height is set
-        (minConfNum > 0 || maxConfNum < Int.MaxValue)    // confirmations are set
-      ) {
-        BadRequest(s"Bad request: both heights and confirmations set")
-      } else {
-        val filteringOpts = if (minHeight == 0 && maxHeight == Int.MaxValue &&
-          minConfNum == 0 && maxConfNum == Int.MaxValue) {
-          None
-        } else if(minHeight > 0 || maxHeight < Int.MaxValue){
-          Some(FilterByHeight(minHeight, maxHeight))
-        } else {
-          Some(FilterByConfirmations(minConfNum, maxConfNum))
-        }
-
+      if (minHeight == 0 && maxHeight == Int.MaxValue && minConfNum == 0 && maxConfNum == Int.MaxValue) {
```

Comment:
BadRequest(s"Bad request: both heights and confirmations set") part seems to be lost